operators/pkg/tenant-controller/tenant_controller_test.go

```diff
@@ -311,6 +311,8 @@ func checkTnClusterResourceCreation(ctx context.Context, tnName, nsName string,
 	Expect(createdRq.Spec.Hard["requests.memory"]).Should(Equal(reqMem))
 	Expect(createdRq.Spec.Hard["count/instances.crownlabs.polito.it"]).Should(Equal(instanceCount))
 
+	By("By checking that the tenant status used resource quotas have been correctly updated")
```

Comment:
Leveraging the resource quota only makes it also easier to add a small test here. Just manually update the resource quota status and assert that the same values are eventually reflected in the tenant status.